proto/viam/app/data/v1/data.proto

```diff
@@ -376,3 +379,12 @@ message BoundingBoxLabelsByFilterRequest {
 message BoundingBoxLabelsByFilterResponse {
   repeated string labels = 1;
 }
+
+message GetDatabaseUserRequest {
+  string org_id = 1;
+}
+
+message GetDatabaseUserResponse {
+  string username = 1; // unsure if we need to repond with username because every org is given one default username
```

Comment:
Thanks for putting these open questions in the comments! @agreenb, two quirks about these API definitions came up as we were implementing. This is the first one; we return the username for the org, but the username can actually just be inferred from the Viam organization ID, so do we need to return this as part of the API? I'm thinking we should just remove it. This would also make it consistent with ConfigureDatabaseUser, which doesn't accept a `username`.

Reply:
If we're always accepting an organization ID and always abstracting the username away from the user, then we shouldn't return it here, and this should instead be GetDatabaseConnection, or something similar.

However, would it be useful for the user to access their underlying username?